zng/docs/spec.md

```diff
@@ -467,7 +494,7 @@ i.e., via `\u{0a}` or `\x0a`.
 A line that begins with `#` is a control message and all other lines
 are values.
 
-### 3.1 ZNG Control Messages
+### 4.1 ZNG Control Messages
```

Comment:
```suggestion
### 4.1 TZNG Control Messages
```

Reply:
@henridf (and cc: @nwt since he gave a 👍  to this comment): Good catch. Upon closer reading though, it seems like there may be more to do here than one surgical change. Throughout the TZNG section, there's places where the spec could probably stand to be explicit in clarifying where we're talking specifically about TZNG vs. general characteristics of ZNG that apply to both types of presentation. This comes up early in the intro text of Section 4 on TZNG: _"A ZNG file/stream is encoded with UTF-8"_ should probably say _"TZNG"_, for instance. If you guys agree, I'll take a shot at making the correct changes throughout that section and ask for a re-review on it.

Reply:
@henridf: I've made the changes to clean up the TZNG section, so PTAL. Not only did I mention TZNG explicitly now in several places, it also led me to make some of the headers more consistent and notice a couple bits that probably belonged in the ZNG section, so I moved those up. I've added new PR comments to call attention to where I did that. I also noticed a couple references to "ZNG type syntax" that I realized should be a link to somewhere, but it's not obvious where, so I'm hoping someone can give me a tip there. That someone could be you. 😄

Reply:
(https://github.com/brimsec/zq/pull/897/files/6d6554fc93539db4e8ee8b4ad87cfc144b465c16#r439974371)